easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.2.eb

```diff
@@ -0,0 +1,46 @@
+
+easyblock = 'PackedBinary'
+
+name = 'sratoolkit'
+version = '2.5.2'
+
+homepage = 'http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software'
+description = """The SRA Toolkit, and the source-code SRA System Development Kit (SDK), will allow you to 
+programmatically access data housed within SRA and convert it from the SRA format"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/%(version)s/']
+sources = ['sratoolkit.%(version)s-ubuntu64.tar.gz']
+
+executables = [
```

Comment:
oh waw... :)

this is going to be a nightmare to maintain, so why not use this instead?

```python
files_to_copy = [('*', 'bin')]
```

If that also copies other cruft into `bin`, we should at least do:

```python
executables = ['abi-dump*', 'abi-load*', ..., 'vdb-*']
```

@verdurin: are you up for giving that a shot, based on this easyconfig?